fraud/interface.go

```diff
@@ -21,6 +21,7 @@ type ProofUnmarshaler func([]byte) (Proof, error)
 type Service interface {
 	Subscriber
```

Comment:
Add to the interface description that Subscriber now errors out if the fraud already exists

Reply:
This Subscribe is not returning an error

Reply:
Ok, we discussed the error return in sync, but the current solution is even better.